src/style.css

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+/* ---------- Big search ------------ */
+
+.big-search {
+  position: relative;
```

Comment:
You could create a mix of classes (e.g. `search big-search` and `search small-search`), class `search` would be responsible for `position: relative` (and other common styles), and classes `big-search` and `small-search` will have more specific styles for each of your fields

Reply:
Thank you, I will take note!